swarmdjango/core/migrations/0001_initial.py

```diff
@@ -0,0 +1,183 @@
+                ('fileName', models.TextField()),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Entry',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('title', models.TextField()),
+                ('text', models.TextField()),
+                ('comments', models.ManyToManyField(to='core.Comment')),
+            ],
+        ),
+        migrations.CreateModel(
+            name='PersonalPage',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('pageType', models.TextField()),
+                ('pageTitle', models.TextField()),
+            ],
+        ),
+        migrations.CreateModel(
+            name='SideBar',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('content', django.contrib.postgres.fields.jsonb.JSONField()),
+            ],
+        ),
+        migrations.CreateModel(
+            name='User',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('username', models.TextField()),
+                ('password', models.TextField()),
+                ('email', models.TextField()),
+                ('firstName', models.TextField()),
+                ('lastName', models.TextField()),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Admin',
+            fields=[
+                ('user_ptr', models.OneToOneField(auto_created=True, on_delete=django.db.models.deletion.CASCADE, parent_link=True, primary_key=True, serialize=False, to='core.User')),
+                ('receiveUpdates', models.BooleanField()),
+            ],
+            bases=('core.user',),
+        ),
+        migrations.CreateModel(
+            name='SponsorPersonalPage',
+            fields=[
+                ('personalpage_ptr', models.OneToOneField(auto_created=True, on_delete=django.db.models.deletion.CASCADE, parent_link=True, primary_key=True, serialize=False, to='core.PersonalPage')),
+                ('missionStatement', models.TextField()),
+                ('reasonForSponsorship', models.TextField()),
+                ('companyLink', models.TextField()),
+            ],
+            bases=('core.personalpage',),
+        ),
+        migrations.CreateModel(
+            name='Wiki',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('title', models.TextField()),
+                ('briefDescription', models.TextField()),
+                ('entries', models.ManyToManyField(to='core.Entry')),
+            ],
+        ),
+        migrations.CreateModel(
```

Comment:
Do we want to have the file upload field added to this or should we worry about file storage fields later? https://docs.djangoproject.com/en/3.1/ref/models/fields/#django.db.models.FileField

Reply:
We could also use the built in image field as well https://docs.djangoproject.com/en/3.1/ref/models/fields/#django.db.models.ImageField

Reply:
Should probably use ImageField, but I think we'll have to look into that later.
We'll need to know where we're storing it/how (mainly for the 'ImageField.upload_to' option)